### PR TITLE
Add a link to instruction for homebrew

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

**Review by PaulWessel — APPROVED** [review]
Should we say the Bundle needs macos > OS 10.15 ?

**Comment by seisman** [comment]
> Should we say the Bundle needs macos > OS 10.15 ?

Yes, we should mention that it only works for macOS >= 10.14.

File: INSTALL.md [hunk 1]
```diff
@@ -73,13 +73,14 @@ Most Linux distributions come with a text editor installed. You may also
 
 ## Mac
 
-There are 3 options for installing GMT 6.1.0 on Mac:
+There are 4 options for installing GMT 6.1.0 on Mac:
 
 1. Use the GMT bundle installer (**recommended**)
```
Comment (seisman) [inline INSTALL.md]:
```suggestion
1. Use the GMT bundle installer (**recommended** for macOS >= 10.14 only) 
```

**Comment by joa-quim** [comment]
But I think it should also be warned that the bundle is **not** recommended for using GMT julia or PyGMT

**Pull request merged**